Fix build failure

On microblaze, we get a failure due to:

   ../attr-2.5.2/libattr/syscalls.c:133:10: error: symver is only supported on ELF platforms

Upstream-Status: Pending

Signed-off-by: Mark Hatle <mark.hatle@amd.com>

Index: attr-2.5.2/libattr/syscalls.c
===================================================================
--- attr-2.5.2.orig/libattr/syscalls.c
+++ attr-2.5.2/libattr/syscalls.c
@@ -31,6 +31,7 @@
  * prefer symver attribute if available (since gcc 10),
  * fall back to traditional .symver asm directive otherwise.
  */
+#ifndef __microblaze__
 #ifdef __has_attribute
 # if __has_attribute(__symver__)
 #  define SYMVER(cn, vn) __typeof(cn) cn __attribute__((__symver__(vn)))
@@ -43,6 +44,9 @@
 			 __asm__(".symver " #cn "," vn)
 # endif
 #endif
+#else
+# define SYMVER(cn, vn)
+#endif
 #ifndef SYMVER
 #  define SYMVER(cn, vn) __asm__(".symver " #cn "," vn)
 #endif
